src/luarocks/cmd/new_version.lua

```diff
@@ -158,6 +161,12 @@ function new_version.command(flags, input, version, url)
       version = flags.tag:gsub("^v", "")
    end
    
+   local out_dir
+   if flags.dir then
+      out_dir = dir.normalize(flags.dir)
+      out_dir = out_dir:match('^(.-)/*$')
```

Comment:
Is this needed after `dir.normalize`? I thought it stripped trailing `/` already